features/providers/public_law_family/merits_child_care_assessment_question_flow.feature

```diff
@@ -103,9 +112,55 @@ Feature: Public law family merits appeal question flow
     And I should see 'Chances of success Completed'
     And I should see 'Assessment of your client Completed'
 
+    #################################
+    # Child care assessment of client
+    # assessment yes, assessment result negative
+    #################################
+    When I click 'Save and continue'
+    Then I should be on the 'check_merits_answers' page showing 'Check your answers'
+    And the govuk-summary-card titled "Assessment of your client" should contain:
+      | question | answer |
+      | Client's ability to care was assessed by the local authority? | Yes |
+      | Assessment result | Positive |
+
+    And I should not see "How the result will be challenged"
+
+    When I click the govuk-summary-card titled "Assessment of your client" Change link
+    Then I should be on the 'assessment_of_client' page showing "Has the local authority assessed your client's ability to care for the children involved?"
+
+    When I click 'Save and continue'
+    Then I should be on the 'assessment_result' page showing "Was the assessment positive or negative?"
+
+    When I choose "Negative"
+    Then I should see "How will the negative assessment be challenged?"
+
+    # Test the error
+    When I click "Save and continue"
+    Then I should see govuk error summary "Enter how the negative assessment will be challenged"
+    And I should not see "Select if the assessment was positive or negative"
+
+    When I fill "proceeding-merits-task-child-care-assessment-details-field-error" with "I will challenge the negative assessment by..."
+    And I click "Save and continue"
+    Then I should be on the 'check_merits_answers' page showing 'Check your answers'
+    And the govuk-summary-card titled "Assessment of your client" should contain:
+      | question | answer |
+      | Client's ability to care was assessed by the local authority? | Yes |
+      | Assessment result | Negative |
+      | How the result will be challenged | I will challenge the negative assessment by... |
+
+    #################################
+    # Child care assessment of client
+    # 4a no
```

Comment:
minor nitpick but should this comment say 'assessment no' like the rest of the test comments?